symbology/apply-unique-values-with-alternate-symbols/src/main/java/com/esri/samples/apply_unique_values_with_alternate_symbols/ApplyUniqueValuesWithAlternateSymbolsSample.java

```diff
@@ -0,0 +1,210 @@
+      SimpleMarkerSymbol simpleBlueSquareSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.SQUARE, 0xFF0000FF, 15);
+      SimpleMarkerSymbol simpleYellowDiamondSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.DIAMOND, 0xFFFFFF00, 15);
+      SimpleMarkerSymbol simpleRedTriangleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, 0xFFFF0000, 15);
+      SimpleMarkerSymbol simplePurpleDiamondSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.DIAMOND, 0xFF800080, 12);
+
+      // convert simple marker symbols to multilayer point symbols
+      MultilayerPointSymbol multilayerBlueSquareSymbol = simpleBlueSquareSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol multilayerYellowDiamondSymbol = simpleYellowDiamondSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol multilayerRedTriangleSymbol = simpleRedTriangleSymbol.toMultilayerSymbol();
+      MultilayerPointSymbol multilayerPurpleDiamondSymbol = simplePurpleDiamondSymbol.toMultilayerSymbol();
+
+      // set scale range values through reference properties for the multilayer symbols
+      multilayerBlueSquareSymbol.setReferenceProperties(new SymbolReferenceProperties(10000.0, 5000.0));
+      multilayerYellowDiamondSymbol.setReferenceProperties(new SymbolReferenceProperties(20000.0, 10000.0));
+      multilayerRedTriangleSymbol.setReferenceProperties(new SymbolReferenceProperties(5000.0, 0.0));
+
+      // create unique value for the red triangle and list of alternate symbols
+      var uniqueValue = new UniqueValueRenderer.UniqueValue("unique values based on request type", "unique value",
+          multilayerRedTriangleSymbol, List.of("Damaged Property"), List.of(multilayerBlueSquareSymbol, multilayerYellowDiamondSymbol));
+
+      // create unique value renderer
+      var uniqueValRenderer = new UniqueValueRenderer(List.of("req_type"), List.of(uniqueValue), "", multilayerPurpleDiamondSymbol);
+
+      // apply default symbol for the unique value renderer.
+      uniqueValRenderer.setDefaultSymbol(multilayerPurpleDiamondSymbol);
+
+      // create a service feature table using the url
+      final var serviceFeatureTable = new ServiceFeatureTable(FEATURE_SERVICE_URL);
+
+      // create a feature layer from the service feature table
+      featureLayer = new FeatureLayer(serviceFeatureTable);
+
+      // set the unique value renderer on the feature layer
+      featureLayer.setRenderer(uniqueValRenderer);
+
+      // check the service feature table has loaded before adding the feature layer to the map
+      serviceFeatureTable.addDoneLoadingListener( () -> {
+        if (serviceFeatureTable.getLoadStatus() == LoadStatus.LOADED) {
+          map.getOperationalLayers().add(featureLayer);
+          vBox.setDisable(false);
+        } else {
+          new Alert(Alert.AlertType.ERROR,
+            "Error loading Feature Table from service" + serviceFeatureTable.getLoadError().getMessage()).show();
```

Comment:
```suggestion
            "Error loading Feature Table from Service: " + serviceFeatureTable.getLoadError().getMessage()).show();
```
Add a space so that the message string isn't just appended directly to your first string.